CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## next
 
+*Enhancements*
+- Officially support Kubernetes 1.14 [#461](https://github.com/Shopify/kubernetes-deploy/pull/461)
```

Comment:
This should also be mentioned in the version compatibility table in the readme

Reply:
I think we only track EoL versions in that table `Last officially supported in gem version`. Though we don't have an entry for 1.9 :(

Reply:
Ah, you're right, my mistake.